fetch.bs

```diff
@@ -4683,6 +4683,7 @@ dictionary RequestInit {
   DOMString integrity;
   boolean keepalive;
   any window; // can only be set to null
+  AbortSignal signal;
```

Comment:
I'm not really sure what you mean by this. Do you mean I should move it just after the `boolean keepalive;` line? How does this make it aligned?